spec/requests/api/projects_spec.rb

```diff
@@ -117,59 +117,34 @@
     end
   end
 
-  describe 'POST /projects' do
-    context 'maximum number of projects reached' do
-      before do
-        (1..user2.projects_limit).each do |project|
-          post api('/projects', user2), name: "foo#{project}"
-        end
-      end
-
-      it 'should not create new project' do
+  describe "POST /projects" do
+    context "maximum number of projects reached" do
+      it 'should not create new project and respond with 403' do
+        User.any_instance.stub(:projects_limit_left).and_return(0)
         expect {
-          post api('/projects', user2), name: 'foo'
+          post api('/projects', user2), name: "foo"
         }.to change {Project.count}.by(0)
+        response.status.should == 403
       end
     end
 
-    it 'should create new project without path' do
-      expect { post api('/projects', user), name: 'foo' }.to change {Project.count}.by(1)
-    end
-
-    it 'should not create new project without name' do
-      expect { post api('/projects', user) }.to_not change {Project.count}
-    end
-
-    it 'should return a 400 error if name not given' do
-      post api('/projects', user)
-      response.status.should == 400
-    end
-
-    it 'should create last project before reaching project limit' do
-      (1..user2.projects_limit-1).each { |p| post api('/projects', user2), name: "foo#{p}" }
-      post api('/projects', user2), name: 'foo'
+    it 'should create new project without path and return 201' do
+      expect { post api("/projects", user), name: 'foo' }.to change {Project.count}.by(1)
```

Comment:
Line is too long. [89/80]<br>Space missing inside {.<br>Space missing inside }.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.